contracts/src/spaces/facets/tipping/ITipping.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// interfaces
+
+// libraries
+
+// contracts
+
+interface ITippingBase {
+  // =============================================================
+  //                           Structs
+  // =============================================================
+
+  struct TipRequest {
+    uint256 tokenId;
+    address currency;
+    uint256 amount;
+    bytes32 messageId;
+    bytes32 channelId;
+  }
+
+  // =============================================================
+  //                           Events
+  // =============================================================
+
+  event Tip(
+    uint256 indexed tokenId,
+    address indexed currency,
+    address sender,
+    address receiver,
+    uint256 amount
+  );
+
+  event TipMessage(bytes32 indexed messageId, bytes32 indexed channelId);
```

Comment:
```suggestion
  event Tip(
    bytes32 indexed channelId,
    uint256 indexed tokenId,
    address indexed currency,
    address sender,
    address receiver,
    uint256 amount,
    bytes32 messageId
  );
```
Is there a benefit of indexing `messageId`?

Reply:
this is for River stream.